deploy-aci/main.tf

```diff
@@ -0,0 +1,95 @@
+#########################################################
+### Defines Azure Resource Manager Terraform Provider ###
+#########################################################
+provider "azurerm" {
+    version = "2.9.0"
+    features {}
+}
+#########################################################
+######### Creates Azure Resource Group ##################
+#########################################################
+resource "azurerm_resource_group" "useast-minecraftaci-rg" {
+  name = var.resource_group_name
+  location = var.resource_location
+}
+
+#########################################################
+######### Creates Azure Storage Account #################
+#########################################################
+
+resource "azurerm_storage_account" "useast-aci-storage" {
+  name = "useastacistorage"
+  resource_group_name = azurerm_resource_group.useast-minecraftaci-rg.name
+  location = azurerm_resource_group.useast-minecraftaci-rg.location
+  account_tier = "Premium"
+  account_replication_type = "LRS"
+  account_kind = "FileStorage"
+
+}
+
+#########################################################
+######### Creates 100GB Azure File Share#################
+#########################################################
+resource "azurerm_storage_share" "useast-aci-share" {
+  name = "acishare"
+  storage_account_name = azurerm_storage_account.useast-aci-storage.name
```

Comment:
if the location is brought in via a parameter and can change, dont hardcode the location name in this resource name